First/src/MediumDifficultyAssignmnets/AnnualSalary.java

```diff
@@ -0,0 +1,42 @@
+package firstPackage.First.src.MediumDifficultyAssignmnets;
+
+public class AnnualSalary {
+    private double totalTax;
+    private int IncomeCases;
+
+    public void findIncomeTax(int taxableIncome) {
+        IncomeCases = (taxableIncome <= 700000) ? 1 : (taxableIncome > 500000 && taxableIncome <= 1000000) ? 2 : (taxableIncome <= 1500000) ? 3 : 4;
```

Comment:
Please correct the condition

Reply:
please check the condition again and change accordingly

Reply:
Hi @prashantkawa 
Done ! Updated Medium Complexity Problem - 1 code as per the conditions.